code/game/gamemodes/shadowling/shadow_abilities.dm

```diff
@@ -0,0 +1,542 @@
+		user.mind.AddSpell(new /obj/effect/proc_holder/spell/blindness_smoke(null))
+
+
+	if(thralls < victory_threshold)
+		to_chat(user, "<span class='shadowling'>You do not have the power to ascend. You require [victory_threshold] thralls, but only [thralls] living thralls are present.</span>")
+
+	else if(thralls >= victory_threshold)
+		to_chat(user, "<span class='shadowling'><b>You are now powerful enough to ascend. Use the Ascendance ability when you are ready. <i>This will kill all of your thralls.</i></span>")
+		to_chat(user, "<span class='shadowling'><b>You may find Ascendance in the Shadowling Evolution tab.</b></span>")
+		for(M in GLOB.alive_mob_list)
+			if(is_shadow(M))
+				var/obj/effect/proc_holder/spell/self/collective_mind/CM
+				if(CM in M.mind.spell_list)
+					M.mind.spell_list -= CM
+					qdel(CM)
+				M.mind.RemoveSpell(/obj/effect/proc_holder/spell/self/shadowling_hatch)
+				M.mind.AddSpell(new /obj/effect/proc_holder/spell/self/shadowling_ascend(null))
+				if(M == user)
+					to_chat(user, "<span class='shadowling'><i>You project this power to the rest of the shadowlings.</i></span>")
+				else
+					to_chat(M, "<span class='shadowling'><b>[user.real_name] has coalesced the strength of the thralls. You can draw upon it at any time to ascend. (Shadowling Evolution Tab)</b></span>") //Tells all the other shadowlings
+
+
+
+///////////////////
+////BLIND SMOKE////
+///////////////////
+
+/obj/effect/proc_holder/spell/blindness_smoke //Spawns a cloud of smoke that blinds non-thralls/shadows and grants slight healing to shadowlings and their allies
+	name = "Blindness Smoke"
+	desc = "Spews a cloud of smoke which will blind enemies."
+	panel = "Shadowling Abilities"
+	charge_max = 600
+	human_req = TRUE
+	clothes_req = FALSE
+	action_icon_state = "black_smoke"
+	action_icon = 'icons/mob/actions/actions_shadowling.dmi'
+	sound = 'sound/effects/bamf.ogg'
+
+/obj/effect/proc_holder/spell/blindness_smoke/cast(mob/living/carbon/human/user) //Extremely hacky
+	if(!shadowling_check(user))
+		revert_cast()
+		return
+	user.visible_message("<span class='warning'>[user] bends over and coughs out a cloud of black smoke!</span>")
+	to_chat(user, "<span class='shadowling'>You regurgitate a vast cloud of blinding smoke.</span>")
+	var/obj/item/reagent_containers/glass/beaker/large/B = new /obj/item/reagent_containers/glass/beaker/large(user.loc) //hacky
+	B.reagents.clear_reagents() //Just in case!
+	B.invisibility = INFINITY //This ought to do the trick
+	B.reagents.add_reagent("blindness_smoke", 10)
+	var/datum/effect_system/smoke_spread/chem/S = new
+	S.attach(B)
+	if(S)
+		S.set_up(B.reagents, 4, 0, B.loc)
+		S.start()
+	qdel(B)
+
+
+
+
+/////////////////////
+////SONIC SCREECH////
+/////////////////////
+
+/obj/effect/proc_holder/spell/unearthly_screech //Damages nearby windows, confuses nearby carbons, and outright stuns silly cones
+	name = "Sonic Screech"
+	desc = "Deafens, stuns, and confuses nearby people. Also shatters windows."
```

Comment:
This ability was meant to counter cyborgs. Since headlamp destruction is now built into Veil, I recommend choosing one or the other; with both cyborgs will become worse than useless - they'll become dead weight.